tracer/src/Datadog.Trace/RemoteConfigurationManagement/Protocol/GetRcmRequest.cs

```diff
@@ -0,0 +1,26 @@
+// <copyright file="GetRcmRequest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+
+namespace Datadog.Trace.RemoteConfigurationManagement.Protocol
+{
+    internal class GetRcmRequest
+    {
+        public GetRcmRequest(RcmClient client, List<RcmCachedTargetFile> cachedTargetFiles)
+        {
+            Client = client;
+            CachedTargetFiles = cachedTargetFiles.Any() ? cachedTargetFiles : null;
```

Comment:
Should we check for null on `cachedTargetFiles`? My understanding from the RFC was that `null` is a valid value for it?

Reply:
Was changed in order to pass system tests, fixed at 10ea17673b64235f948ebde1f87ba0bee0454791